ui-react/packages/atlasmap-ui/src/AtlasmapUI/MappingDetails/MappingField.tsx

```diff
@@ -16,6 +16,17 @@ import {
 } from '@patternfly/react-core';
 import React, { FunctionComponent, useState, Children } from 'react';
 import { BoltIcon, InfoAltIcon, TrashIcon } from '@patternfly/react-icons';
+import { css, StyleSheet } from '@patternfly/react-styles';
+
+const styles = StyleSheet.create({
+  dataListContent: { boxShadow: 'none' },
```

Comment:
personal opinion - I like to have a box shadow even for empty content - it gives an idea what's in the canvas